OxfordBodleian/Juel-Jensen/BDLaethf29.xml

```diff
@@ -182,8 +182,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <add place="overstrike">ገብረ፡</add>
                                     </subst>
                                     <hi rend="rubric">ማርያም፡</hi></persName>ወለጸሐፊሁ፡ <hi rend="rubric"
-                                    >ገብረ፡ <del rend="erasure" unit="chars" extent="4"><certainty
-                                            locus="value" match="../@extent" assertedValue="5"
+                                    >ገብረ፡ <del rend="erasure" unit="chars" quantity="4"><certainty
```

Comment:
In this case, I did not change the attribute extent to quantity since here I am not sure about the exact number of erased characters. Here we have a range of 4-5 characters. 
I understood that quantity is only valid in cases when we know the exact number of lost characters.
In any case, I  think I am also currently updating the file (I can check only the next time I will be in the office). Would it be okay that I myself would include those changes (if we change it) together with my other updates? I might be in the office on Wendesday, so I know wehther or not I committed already some changes to this file.
Thank you!!

Reply:
@thea-m : you are right, the encoding at this place also needs to be adjusted. I must have overseen it wghen I adjusted all other occurences. Thanks for spotting it. I wold need to do some more corrections anyway. Would it be ojay if I include all in my file, you you could delete it here?

Reply:
Yes, I had already deleted it in https://github.com/BetaMasaheft/Manuscripts/pull/878/commits/0af6bff8315b9e4da3fbdb91315c117082c8815d, it should be as before now